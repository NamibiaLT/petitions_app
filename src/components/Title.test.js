import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Title from './Title';
import React from 'react';


describe('Title', () => {
  it('should render a correct text from props', () => {
    const text = "Something";
    const component = shallow(<Title text={text} />);

    expect(component).toMatchSnapshot();
  });
});
