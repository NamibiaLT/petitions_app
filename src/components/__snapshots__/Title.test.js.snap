// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render a correct text from props 1`] = `
<h1
  className="title"
>
  Something
</h1>
`;
